### PR TITLE
Fix syntax error in Pipe example

### PR DESCRIPTION
**Pull request by antonblanchard**

### CODE REVIEW

**Comment by jackkoenig** [comment]
@antonblanchard can you sign the CLA?

**Comment by seldridge** [comment]
Friendly ping @antonblanchard.

**Comment by nullobject** [comment]
I think we could go one step further with the fix, the name should be `pipe`:

```scala
 val pipe = Module(new Pipe(UInt(8.W), 4))
```

File: src/main/scala/chisel3/util/Valid.scala [hunk 1]
```diff
@@ -152,7 +152,7 @@ object Pipe {
   * As an example, to construct a 4-stage pipe of 8-bit [[UInt]]s and connect it to a producer and consumer, you can use
   * the following:
   * {{{
-  *   val foo = Module(new Pipe(UInt(8.W)), 4)
+  *   val foo = Module(new Pipe(UInt(8.W), 4))
```
Comment (nullobject) [inline src/main/scala/chisel3/util/Valid.scala:155]: I think the value should actually be named `pipe`:

```suggestion
  *   val pipe = Module(new Pipe(UInt(8.W), 4))
```

Comment (jackkoenig) [reply src/main/scala/chisel3/util/Valid.scala:155]: 👍

**Pull request closed without merging**